instrumentation/redisson-3.0/redisson-3.0.gradle

```diff
@@ -0,0 +1,23 @@
+// Set properties before any plugins get loaded
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.redisson"
+    module = "redisson"
+    versions = "[3.0.0,)"
+    assertInverse = true
+  }
+}
+
+dependencies {
+  compileOnly group: 'org.redisson', name: 'redisson', version: '3.0.0'
+
+  testLibrary group: 'org.redisson', name: 'redisson', version: '3.0.0'
+  testImplementation group: 'com.github.kstyrc', name: 'embedded-redis', version: '0.6'
+  latestDepTestLibrary group: 'org.redisson', name: 'redisson', version: '3.+'
```

Comment:
it looks like `3.+` resolves the same as `+`, so using the new `library` dependencies, you don't need this (it will default to `+`)
```suggestion
```

Reply:
Thanks. fixed at lastest commit.